### ISSUE TITLE
Mesh Quality is bad

### ISSUE THREAD

**Issue by abhishekmonogram** [19reborn/NeuS2#17]
Hello, 

Thank you so much for releasing the code. I try to reconstruct using my own set of images and the reconstructed mesh looks bad. I have tried other NSR techniques(eg. neuralangelo and instant-nsr) with the same set of images and the reconstructions are good. I was wondering if there are any hyperparameters that I can tune to make my mesh output better?

**Comment by erin47** [comment]
Hi,
You can adjust the hyperparameters in the configuration file, such as the learning rate https://github.com/19reborn/NeuS2/blob/055b39b9d4f76ddd6ef811ad5519426b6c8dd8ad/configs/nerf/base.json#L15 and eikonal loss weight https://github.com/19reborn/NeuS2/blob/055b39b9d4f76ddd6ef811ad5519426b6c8dd8ad/configs/nerf/base.json#L129.

**Comment by abhishekmonogram** [comment]
Thank you. I will play around with those. Is there a way for me to use GUI
to visualise the sdf or the mesh while it is being created? Just like
instant-ngp’s density field . Right now there is no way for me to know if
my model converges. I have to wait till I finish training. A visualisation
while it is training can save me a lot of time

**Comment by erin47** [comment]
Of course, within GUI, you can open the "Marching Cubes Mesh Output" column and click "Mesh it!" to visualize the mesh result while training.

**Comment by abhishekmonogram** [comment]
@erin47 thank you so much. I don't see any terminal commands to use the GUI. I couldn't see any documentation. Can you please point me to resource where I can learn more about using the GUI?

**Comment by abhishekmonogram** [comment]
@erin47 Thank you. Despite changing the hyperparams you suggested and some more like hash grid size and num levels, the model isn't still converging and creating a mesh in my case. I tried the synthetic data and it works really well. Any other suggestions? I think my data is good enough to generate meshes in off the shelf softwares.

**Comment by 19reborn** [comment]
Hi @abhishekmonogram, is it possible for you to share us the data? I think the problem is in the coordinate system or the geometry initilization strategy.

**Comment by abhishekmonogram** [comment]
Sure I can. Where do you want me to share it with you?

**Comment by 19reborn** [comment]
Could you please put it in the google drive and then send the link to my email `w752531540@gmail.com`?

**Comment by dFuong** [comment]
I have an question "Can run with single view input" .

**Comment by 19reborn** [comment]
Hi @dFuong, NeuS2 requires dense multi-view images to reconstruct good geometry.
